PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -559,26 +559,25 @@ protected function wait_channel($channel_id, $timeout = 0)
                 }
                 continue;
 
-            } else {
```

Comment:
Yes it jumps to the next iteration -> so it continues with the next element on the top of the `while` .

That's why you need no else at all.